### PR TITLE
C2LC-490: Announce scene name in scene announcements.

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [C2LC-490](https://issues.fluidproject.org/browse/C2LC-490) for details.

### CODE REVIEW

**Comment by chosww** [comment]
The announcement should include which world the character is in, I suggest making changes to the beginning of the scene announcement like:

"Scene, in deep ocean", "Scene, in jungle safari" .. etc

**Comment by the-t-in-rtf** [comment]
I did a bunch of manual testing of this, as far as I can see all announcements are as discussed both when manually announcing the scene, and when using VoiceOver.

File: src/Scene.js [hunk 1]
```diff
@@ -201,7 +204,9 @@ class Scene extends React.Component<SceneProps, {}> {
                     numColumns,
                     numRows,
                     direction,
-                    relativeDirection: this.getRelativeDirection(xPos, yPos)
+                    relativeDirection: this.getRelativeDirection(xPos, yPos),
+                    world: this.props.world,
```
Comment (sbates-idrc) [inline src/Scene.js:208]: Please localize the world, rather than using the internal name directly.

File: src/Scene.js [hunk 2]
```diff
@@ -216,7 +221,9 @@ class Scene extends React.Component<SceneProps, {}> {
                         xPos: columnLabel,
                         yPos: rowLabel,
                         direction,
-                        item: itemOnGridCell
+                        item: itemOnGridCell,
+                        world: this.props.world,
```
Comment (sbates-idrc) [inline src/Scene.js:225]: Please localize the world, rather than using the internal name directly.

File: src/Scene.js [hunk 3]
```diff
@@ -227,7 +234,9 @@ class Scene extends React.Component<SceneProps, {}> {
                     numRows: this.props.dimensions.getHeight(),
                     xPos: columnLabel,
                     yPos: rowLabel,
-                    direction
+                    direction,
+                    world: this.props.world,
```
Comment (sbates-idrc) [inline src/Scene.js:238]: Please localize the world, rather than using the internal name directly.

**Comment by the-t-in-rtf** [comment]
Thanks for catching and fixing the raw world name issue.

**Pull request merged**